src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/params.py

```diff
@@ -0,0 +1,58 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import json
+from azure.cli.command_modules.lab.validators import (validate_authentication_type, validate_image_argument,
+                                                      validate_network_parameters, validate_location)
+
+from ._util import ParametersContext
+
+
+with ParametersContext(command='lab vm create') as c:
+    c.register_alias('resource_group', ('--resource-group', '-g'))
+
+    # Authentication related arguments
+    authentication_group_name = 'Authentication'
+    c.argument('admin_username', arg_group=authentication_group_name)
+    c.argument('admin_password', arg_group=authentication_group_name)
+    c.argument('authentication_type', arg_group=authentication_group_name, validator=validate_authentication_type)
+    c.argument('ssh_key', arg_group=authentication_group_name)
+    c.argument('generate_ssh_keys', action='store_true', arg_group=authentication_group_name)
+    c.ignore('is_authentication_with_ssh_key')
+    c.ignore('user_name')
+    c.ignore('password')
+
+    # Add Artifacts from json object
+    c.register('artifacts', ('--artifacts',), type=json.loads)
```

Comment:
You don't need to specify the options list if it is the same as the dest (which this definitely is).